gengokit/middlewares/templates/endpoints.go

```diff
@@ -4,21 +4,21 @@ const EndpointsBase = `
 package middlewares
 
 import (
-	"github.com/go-kit/kit/endpoint"
 	svc "{{.ImportPath -}} /generated"
 )
 
-// WrapEndpoint will be called individually for all endpoints defined in
-// the service. Implement this with the middlewares you want applied to
-// every endpoint.
-func WrapEndpoint(in endpoint.Endpoint) endpoint.Endpoint {
-	return in
-}
-
 // WrapEndpoints takes the service's entire collection of endpoints. This
 // function can be used to apply middlewares selectively to some endpoints,
 // but not others, like protecting some endpoints with authentication.
```

Comment:
This documentation becomes incomplete if the user adds arguments to the `in.WrapAll` call. Here's a wag at an improvement.

> WrapEndpoints accepts the service's entire collection of endpoints, so that a set of middlewares can be wrapped around every middleware (e.g., access logging and instrumentation), and others wrapped selectively around some endpoints and not others (e.g., endpoints requiring authenticated access).